azuread/table_azuread_conditional_access_named_location.go

```diff
@@ -0,0 +1,163 @@
+package azuread
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/iancoleman/strcase"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+
+	msgraphcore "github.com/microsoftgraph/msgraph-sdk-go-core"
+	"github.com/microsoftgraph/msgraph-sdk-go/identity"
+	"github.com/microsoftgraph/msgraph-sdk-go/models"
+)
+
+//// TABLE DEFINITION
+
+func tableAzureAdConditionalAccessNamedLocation(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "azuread_conditional_access_named_location",
+		Description: "Represents an Azure Active Directory (Azure AD) Conditional Access Named Location.",
+		Get: &plugin.GetConfig{
+			Hydrate: getAdConditionalAccessNamedLocation,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isIgnorableErrorPredicate([]string{"Request_ResourceNotFound", "Invalid object identifier"}),
+			},
+			KeyColumns: plugin.SingleColumn("id"),
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAdConditionalAccessNamedLocations,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isIgnorableErrorPredicate([]string{"Request_UnsupportedQuery"}),
```

Comment:
I'm sorry I don't think I fully understand you. I copied this part from the other tables in the plugin. I changed the type column to location_type because type is a reserved in SQL. I can't think about a specific query parameter combinations where we encounter the Request_UnsupportedQuery.
Added support for id, location_type to be handled by our table code. Is it alright now?